spec/feature/integration_spec.rb

```diff
@@ -1312,6 +1326,274 @@
 	visit foreign_courses_tamu_courses_path
 	expect(page).to have_content('Software Engineering')
 	
-	
   end
 end
+
+RSpec.describe 'Creating request cases', type: :feature do
```

Comment:
This looks good. Good job breaking up the tests, too.

Reply:
Thanks!